src/test/javascript/spec/component/course/course-exercises.component.spec.ts

```diff
@@ -103,28 +96,28 @@ describe('CourseExercisesComponent', () => {
     });
 
     afterEach(() => {
-        sinon.restore();
-        jest.clearAllMocks();
+        jest.restoreAllMocks();
     });
 
     it('should initialize', () => {
-        expect(component.course).to.deep.equal(course);
-        expect(courseCalculationSpy.mock.calls.length).to.equal(2);
-        expect(courseCalculationSpy.mock.calls[0][0]).to.eq(course.id);
-        expect(courseCalculationSpy.mock.calls[1][0]).to.eq(course.id);
+        expect(component.course).toEqual(course);
+        expect(courseCalculationSpy.mock.calls.length).toEqual(2);
+        expect(courseCalculationSpy.mock.calls[0][0]).toEqual(course.id);
```

Comment:
Is it possible to use `toBe()` here? I would assume because the id should be a number. If yes, try to do it that way.
```suggestion
        expect(courseCalculationSpy.mock.calls[0][0]).toBe(course.id);
```